resources/measures/advanced_rtu_control/measure.rb

```diff
@@ -0,0 +1,335 @@
+      if dsn_oa.outdoorAirMethod == 'Maximum'
+        sum_oa_rate += [oa_for_people, oa_for_floor_area, oa_rate, oa_for_volume].max
+      elsif dsn_oa.outdoorAirMethod == 'Sum'
+        sum_oa_for_people += oa_for_people
+        sum_oa_for_floor_area += oa_for_floor_area
+        sum_oa_rate += oa_rate
+        sum_oa_for_volume += oa_for_volume
+      end
+    end
+
+    tot_oa_flow_rate += sum_oa_for_people
+    tot_oa_flow_rate += sum_oa_for_floor_area
+    tot_oa_flow_rate += sum_oa_rate
+    tot_oa_flow_rate += sum_oa_for_volume
+
+    # Convert to cfm
+    tot_oa_flow_rate_cfm = OpenStudio.convert(tot_oa_flow_rate, 'm^3/s', 'cfm').get
+
+    #runner.registerInfo("For #{thermal_zone.name}, design min OA = #{tot_oa_flow_rate_cfm.round} cfm.")
+
+    return tot_oa_flow_rate
+  end
+
+  # define what happens when the measure is run
+  def run(model, runner, user_arguments)
+    super(model, runner, user_arguments)  # Do **NOT** remove this line
+
+    # use the built-in error checking
+    if !runner.validateUserArguments(arguments(model), user_arguments)
+      return false
+    end
+	#read in arguments
+	add_econo = runner.getBoolArgumentValue('add_econo', user_arguments)
+	add_dcv = runner.getBoolArgumentValue('add_dcv', user_arguments)
+	
+	#Set constants
+	fan_tot_eff = 0.7 #Eplus default, reasonable
+    fan_mot_eff = 0.9 #Eplus default, reasonable 
+	
+	#set airflow design ratios
+	max_flow = 0.9
+	min_flow = 0.4 
+	
+	min_flow_fraction = 0.3 #typ for non-inverter driven motors 
+	
+	
+	#Sizing run 
+	standard = Standard.build('90.1-2013')
+    if standard.model_run_sizing_run(model, "#{Dir.pwd}/advanced_rtu_control") == false
+      runner.registerError('Sizing run for Hardsize model failed, cannot hard-size model.')
+      puts('Sizing run for Hardsize model failed, cannot hard-size model.')
+      puts("directory: #{Dir.pwd}")
+      return false
+    end
+
+    #apply sizing values
+    model.applySizingValues
+	
+
+   	model.getAirLoopHVACs.sort.each do |air_loop_hvac|
+	    if air_loop_hvac_unitary_system?(air_loop_hvac) #applying to unitary systems 
+		   #set control type
+			air_loop_hvac.supplyComponents.each do |component|#more efficient way of doing this? 
+				obj_type = component.iddObjectType.valueName.to_s
+			    case obj_type
+                when 'OS_AirLoopHVAC_UnitarySystem'
+					component = component.to_AirLoopHVACUnitarySystem.get
+					component.setControlType('SingleZoneVAV') 
+					#Set overall flow rates for air loop 
+					if air_loop_hvac.autosizedDesignSupplyAirFlowRate.is_initialized
+						des_supply_airflow = air_loop_hvac.autosizedDesignSupplyAirFlowRate.get
+						#puts ("des supply airflow" + "#{air_loop_hvac.name.to_s}" "#{des_supply_airflow}" + "new max" + "#{max_flow*des_supply_airflow}")
+						component.setSupplyAirFlowRateDuringCoolingOperation(max_flow*des_supply_airflow) #Set max based on limit after retrofit 
```

Comment:
I have concerns over using this 90% factor for meeting loads. If the system and the coils are sized based on 100% design airflow, wouldn't we get unmet hours by limiting the airflow to 90%?

Reply:
Per our discussion, I've revised this to full airflow.